portmap/core/track.py

```diff
@@ -0,0 +1,18 @@
+from functools import wraps
+
+from .models import TrackArticleView
+
+def track_view(function):
```

Comment:
I like this idea in some ways.  However, making this a decorator implies that we would decorate possibly more than one view with the same decorator.  If that is the case, a decorator is a great reusable modular approach.  However if this really only works with the article view, then it's not so reusable and the complication of using a decorator was unnecessary.  

Also we don't need to make this a generic view tracker - we specifically want numbers for article views.  Making it a generic view tracker means we have to figure out what article each path refers to.  If we ever change the request path, or if the request path changes due to somebody capitalizing a link , it would no longer track the same thing.